ui/src/app/dashboard/services/DashService.js

```diff
@@ -0,0 +1,32 @@
+(function(){
+  'use strict';
+
+  angular.module('app')
+        .service('DashService', [
+        '$http',
+      dashService
+  ]);
+
+  function dashService($http){
+    var addDownload = function(link) {
+      return $http({
+          method: 'POST',
+          url: 'http://localhost:5000/api/download',
```

Comment:
Create an const value for the base url. May be an anglar value? And refer it here.
`url: BassaUrl+ 'api/download'`

easier to configure when moving to production. Or else will have to change this value in number of places.